Add ISC access node guide (#1562)
iotaledger/iota-wiki: docs/maintain/wasp/v1.0.0-rc.6/docs/how-tos/running-a-node.md

## Set Up

Follow the instructions on the [hornet setup](/hornet/how_tos/using_docker/).

:::note

> This is aimed at production-ready deployment. If you're looking to spawn a local node for testing/development, please see the [IOTA Sandbox](/iota-sandbox/welcome)

Reviewer: Just an alternative suggestion:

```suggestion
This how-to is meant to be used for a production-ready deployment. For local testing or development, please see the [IOTA Sandbox](/iota-sandbox/welcome).
```
